tests/test_forms.py

```diff
@@ -87,18 +63,23 @@ def test_compare_money_field_with_taxed_money_field():
     assert field_1 == field_2
 
 
-def test_money_field_instance_amount_and_currency_are_set_separately():
-    instance = Model(price_net_amount=Decimal("10"), currency="USD")
-    assert instance.price_net.amount == Decimal("10")
-    assert instance.price_net.currency == "USD"
-    assert instance.price_net == Money("10", "USD")
+def test_render_money_input():
+    widget = widgets.MoneyInput(choices=[("USD", "US dollar")], attrs={"key": "value"})
+    result = widget.render("price", Money(5, "USD"), attrs={"foo": "bar"})
+    assert 'foo="bar"' in result
+    assert 'name="price_0"' in result
+    assert 'name="price_1"' in result
+    assert 'key="value"' in result
+    assert 'value="5"' in result
+    assert "USD" in result
```

Comment:
Perhaps this could be tested with snapshot?

Also consider changing tests names to descriptive ones, eg `test_money_input_widget_renders`

Reply:
Not sure about adding snapshottest for one or two tests. I would say it's `good enough`.

Reply:
https://github.com/mirumee/django-prices/issues/80